compose-shadow/build.gradle.kts

```diff
@@ -73,7 +73,6 @@ android {
 
     defaultConfig {
         minSdk = libs.versions.androidMinSdk.get().toInt()
-        lint.targetSdk = libs.versions.androidTargetSdk.get().toInt()
```

Comment:
The target SDK shouldn't be removed.

However, `lint.targetSdk` is an error, it only affects the linter. I fixed many instances of this in another PR, guess this one went unnoticed :D

This should be
```kotlin
targetSdk = libs.versions.andtoidTargetSdk.get().toInt()
```

Reply:
Yes, with your help, the project has been standardized a lot💗. I will modify this.

Reply:
> The target SDK shouldn't be removed.
> 
> However, `lint.targetSdk` is an error, it only affects the linter. I fixed many instances of this in another PR, guess this one went unnoticed :D
> 
> This should be
> 
> ```kotlin
> targetSdk = libs.versions.andtoidTargetSdk.get().toInt()
> ```

```
'var targetSdk: Int?' is deprecated. Will be removed from library DSL in v9.0. Use testOptions. targetSdk or/ and lint. targetSdk instead.
```

I will restore lint.targetSdk.

Reply:
Glad I was of help! It's been fun to work on this, my first involvement in an OSS project as a collaborator 😁